web/src/components/modals/AutomaticUpdatesModal.jsx

```diff
@@ -214,18 +249,42 @@ export default class AutomaticUpdatesModal extends React.Component {
           </div>
           <div className="flex-column flex1 u-marginTop--15">
             <p className="u-fontSize--normal u-textColor--primary u-fontWeight--bold u-lineHeight--normal">Automatically deploy new versions</p>
-            <span className="u-marginTop--5 u-marginBottom--15 u-fontSize--small u-textColor--info u-fontWeight--medium">Releases without a valid <a href="https://semver.org/" className="replicated-link" target="_blank" rel="noopener noreferrer">semantic version</a> will <span className="u-fontWeight--bold">not</span> be automatically deployed.</span>
-            <Select
-              className="replicated-select-container flex1"
-              classNamePrefix="replicated-select"
-              placeholder="Automatically deploy new versions"
-              options={SEMVER_AUTO_DEPLOY_OPTIONS}
-              isSearchable={false}
-              getOptionValue={(option) => option.label}
-              value={selectedSemverAutoDeploy}
-              onChange={this.handleSemverAutoDeployOptionChange}
-              isOptionSelected={(option) => { option.value === selectedSemverAutoDeploy }}
-              />
+            { semVerEnabled ?
```

Comment:
how about having the variable names be consistent? `semVerEnabled` vs `isSemverRequired`

Reply:
i would've thought they're two different things, but it seems like they're the same?

Reply:
Yes, they are the same. I will change to `isSemverRequired`